db/schema.rb

```diff
@@ -334,7 +338,6 @@
     t.string "school"
     t.string "major"
     t.string "year"
-    t.string "theme", default: "default"
```

Comment:
The theme field was erroneously introduced into the users schema based on https://github.com/autolab/Autolab/pull/1234/files#diff-cbf8b25d6d853acf58fa9057841f407d29ffe90649c75cf9e3f51b2d6e3aa1d3R337. I believe the reason was because Xinyi ran the migrations for https://github.com/autolab/Autolab/pull/1139/files#diff-e625c8fa2a7f730fcafe9acb6d8bf1c1fba45511b896c86185cf4d06c0bc354eR3 when reviewing the PR and committed the schema as well by accident. But in any case, this will remove this field